docs/Server/API/V1/contents.md

```diff
@@ -126,7 +126,7 @@ The `type` codes are:
 {
     "name": "reference-result",
     "type": "sym",
-    "uri": "http://hostname/api/v1/datasets/contents/<id>/sample1"
+    "uri": "http://hostname/api/v1/datasets/<id>/contents/linkresult"
 }
```

Comment:
Line 129 does not seem to match line 127...and that's because the URI is referring to the referent of the symlink named at line 127?  It might be helpful if there were a `"type": "dir"` example here, as well (perhaps for `linkresult`).

The text says that symlinks to directories are listed in the `"directories"` list...should it close the loop and say that other symlinks are listed in the `"files"` list?

At line 147, we talk about symlinks which refer to files within the dataset...we don't comment about symlinks which either point outside it or point to non-existent files (or to directories)...should we?

Reply:
Oh, right...I'd forgotten about the `reference-result` symlinks...I think it's sort of important that they "work".  😒

As for `libvirt`...I seem to recall us having some failures/weird behavior around `libvirt`...I wonder if that's related to this.  🤔

Anyway, it sounds like we are done here for now, so I shall resolve.